crates/juniper-axum/src/relay/connection.rs

```diff
@@ -27,8 +27,8 @@ where
     }
 
     pub fn build_connection(nodes: Vec<Node>, first: Option<usize>, last: Option<usize>) -> Self {
-        let has_next_page = first.map_or(false, |first| nodes.len() > first);
-        let has_previous_page = last.map_or(false, |last| nodes.len() > last);
+        let has_next_page = first.is_some_and(|first| nodes.len() > first);
+        let has_previous_page = first.is_some_and(|first| first > 0);
```

Comment:
This will return true even for the first page, while it should actually being false.

Reply:
The first element has ID 1?

Reply:
first is not offset, it means pick `first` cnt of elements after `cursor`